community/examples/remote-desktop.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: remote-desktop
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: remote-desktop
+  region: us-east4
+  zone: us-east4-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network1
+    source: modules/network/vpc
+    kind: terraform
+
+  - id: remote-desktop
+    source: community/modules/remote-desktop/chrome-remote-desktop
+    use: [network1]
+    settings:
+      instance_count: 1
+      install_nvidia_driver: true      # Must be set to true accept Nvidia ToS to install drivers
```

Comment:
Here a thought crossed my mind, if we set install_nvidia_driver: false, and guest_accelerator=[], do we still get a fully functional CRD machine without GPU acceleration?

Reply:
I have opened a follow on task for this.